api/src/main/resources/application.properties

```diff
@@ -31,3 +31,9 @@ logging.level.liquibase=WARN
 ## -----------------------------------------------------------------------------------------------------  Health Checks
 health.requiredSpareMemoryInMB=32
 health.urlsToCheck=http://www.google.com,http://www.facebook.com
+
+## -----------------------------------------------------------------------------------------------------  TLS
+server.ssl.key-store-type=PKCS12
```

Comment:
We should probably add `server.port=8443` here as well to be explicit.

Reply:
Will that change the port to 8443 in prod or will something override it to be 443?

Reply:
Correct. We're fine with 8443 in Prod since we are behind a balancer anyway.